.github/workflows/wheels.yml

```diff
@@ -269,6 +255,23 @@ jobs:
         name: dist-sdist
         path: dist/*.tar.gz
 
+  scientific-python-nightly-wheels-publish:
+    if: github.event_name == 'schedule' || github.event_name == 'workflow_dispatch'
+    needs: [build-2-native-wheels, windows]
+    runs-on: ubuntu-latest
+    name: Upload release to PyPI
```

Comment:
```suggestion
    name: Upload release to scientific-python-nightly-wheels
```
?

Reply:
Thanks for the suggestion @nulano. I took the other one since it was fixing the "release" -> "wheels" issue as well.